src/storageSignals.js

```diff
@@ -8,7 +8,14 @@ createEffect(() => {
     const token1_in_url = urlParams.get('token1');
     const app_id_in_url = urlParams.get('app_id');
     const endpoint_in_url = urlParams.get('server_url');
+    const is_production = window.location.hostname === 'api.deriv.com';
+    localStorage.setItem('app_id', 31063);
+    localStorage.setItem('server_url', 'https://blue.binaryws.com');
```

Comment:
when localStorage is set in the app then we should use it instead of setting it to default. We need to move this code to if statement to set default only when localstorage app_id or server_url is not set